fabric-lifecycle-events-v1/src/main/java/net/fabricmc/fabric/api/event/lifecycle/v1/ServerEntityEvents.java

```diff
@@ -75,6 +78,17 @@ private ServerEntityEvents() {
 		}
 	});
 
+	/**
+	 * Called during {@link LivingEntity#tick()} if the Entity's equipment has been changed.
```

Comment:
This documentation could clarify 'changed' as using the `ItemStack.areEqual` method to compare item, count, and NBT.

Reply:
Im not too sure this is that useful to know tbh, and might be best kept as an implimentation detail (In code that isnt ours) rarther than something defined in API.

Reply:
I think 'changed' is too vague for a low-level event like this one. From just reading the summary of this event, I thought that this event would only be invoked if equipment was replaced with another stack entirely. Instead, this event is invoked if any equipment stack mutates at all.